docs/contributing/plugins/README.md

```diff
@@ -28,26 +29,62 @@ basicPluginBuilder(
     # You can add more names after the inner column if multiple plugins use
     # this layout
     ('My plugin name',),
-    # A list of parameter indexes
-    [45, 46, 35, 37, 218, 219, 220, 221],
-    # A color to represent the parameters (this can also be a list)
-    Color.fromInteger(0x206cc8)
+    # A list of parameter indexes - these will map left to write to the 
+    # relevant faders on your device, so in this example tuning will be
+    # fader 1, Waveform will be fader 2, Cutoff will be Fader 3, Resonance
+    # will be Fader 4 and so on.
+    [
+    0,  # Tuning
+    1,  # Waveform
+    2,  # Cutoff
+    4,  # Resonance
+    5,  # Envelope Modulation
+    6,  # Decay
+    7,  # Accent
+    8,  # Volume
+    ]
+    # A color to represent the parameters (this can also be a list, to do so you
+    # would need to list the colour parameters in square brackets with commas between
+    # each parameter - ie ([0x206cc8,0x222222,0x888888,0x206cc8,0x222222])
+    
+    Color.fromInteger(0x206cc8) # this would map all parameters to a single color
+    
 )
 ```
 
-## Creating a Plugin: The Hard (But More Powerful) Way
+## Creating a Plugin Binding: The Hard (But More Powerful) Way
+Plugins can also be bound manually to a [`DeviceShadow`](device_shadow.md)
+object, which represents the plugin's own private copy of the device being
+mapped to. This can be done one of two ways:
+
+* Binding manually - this will be covered in more detail later
+
+* Using <a href="https://github.com/MiguelGuthridge/Universal-Controller-Script/blob/main/docs/contributing/plugins/mapping_strategy.md">mapping strategies</a> given as arguments to the 'super' constructor. 
+  
+  * super().__init__(shadow, [PedalStrategy()]) for example, would map your
+  pedal input automatically to the VST it's applied to.
+  
+  * super().init(shadow, [SimpleFaders(0,1,2,4,5,6,7,8, colors=(0x206cc8))])
+  would be equivalent to the "basicPluginBuilder" function example above
+  
+  Consult ['Mapping Strategy'](mapping_strategy.md) for more options.
+  
 
-When a plugin is created, it should bind callback functions to a
-[`DeviceShadow`](device_shadow.md) object, that represents the plugin's own
-private copy of the device that is being mapped to. This can either be done
-manually, or with [mapping strategies](mapping_strategy.md), given as arguments
-to the `super` constructor. Callbacks can be decorated using
-[event filters](filters.md) to filter out unwanted events.
+Callbacks can be also be decorated using [event filters](filters.md)
+to filter out unwanted events. For example:
+
+    @filterToGeneratorIndex() # Filter out plugins when the active plugin isn't a generator
+    def eventCallback(self, control: ControlShadowEvent, index: GeneratorIndex, *args: Any) -> bool:
+        # Set the parameter
+        plugins.setParamValue(control.value control.coordinate[1], *index)
+        # Handle the event
+        return True
```

Comment:
I'd prefer a consistent style of code blocks. Could you unindent this, then wrap it in triple backticks (```) to keep it consistent with code elsewhere in the project?